### ISSUE TITLE
frame not masked so close the connection for a big packet size.

### ISSUE THREAD

**Issue by PNixx** [walkor/Workerman#527]
In 3.5.25 version I send 200kb binary data and receive 200kb data. But After update to 3.5.26 version I see error:
```
frame not masked so close the connection
```
Check data size send and received:
```
//send: 189388
//received: 132450
```

I use nginx proxy with config:
```
        location /ws/ {
                proxy_pass http://localhost:8190/;

                proxy_http_version 1.1;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";

                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header Host $http_host;
        }
```
Without nginx proxy v3.5.26 correctly works.

Release diff: https://github.com/walkor/Workerman/compare/v3.5.25...v3.5.26

**Comment by walkor** [comment]
It is a bug for v3.5.26~v3.5.28. Please use v3.5.29 or greater version.

**Comment by PNixx** [comment]
I try last version 4.+ and see the same problem

**Comment by walkor** [comment]
Fixed in version v3.5.30 and v4.0.3. Please upgrade.
